tests/test_primitive_play.py

```diff
@@ -1,6 +1,5 @@
 import pytest
-from random import shuffle
-from src.euchre.primitive_play import create_deck, create_players
+from euchre.primitive_play import create_deck, create_players
```

Comment:
The `euchre` package is installed (and is a real python package) so you don't need to (and shouldn't) import it via it's relative path.